standards_lab/api/views.py

```diff
@@ -116,7 +118,10 @@ def delete(self, *args, **kwargs):
         upload_type_key = "%sFiles" % file_info["uploadType"]
 
         path = os.path.join(
-            settings.ROOT_PROJECTS_DIR, project["name"], file_info["fileName"]
+            settings.ROOT_PROJECTS_DIR,
+            project["name"],
+            ("datas" if file_info["uploadType"] == "data" else "schemas"),
```

Comment:
As `uploadType` key can be any string from the user it would be better to handle this either fully https://github.com/OpenDataServices/standards-lab/blob/main/standards_lab/api/views.py#L182 with an exception or not at all and allow an exception to be handled later on

Reply:
That if statement added to make sure correct type has been passed